playground/backend/internal/emulators/kafka.go

```diff
@@ -31,43 +35,90 @@ import (
 )
 
 const (
-	brokerCount        = 1
 	addressSeperator   = ":"
 	pauseDuration      = 100 * time.Millisecond
-	globalDuration     = 2 * time.Second
+	globalDuration     = 120 * time.Second
 	bootstrapServerKey = "bootstrap.servers"
 	networkType        = "tcp"
 	jsonExt            = ".json"
 	avroExt            = ".avro"
 )
 
 type KafkaMockCluster struct {
-	cluster *kafka.MockCluster
-	host    string
-	port    string
+	cmd                *exec.Cmd
+	host               string
+	port               string
+	preparerParameters map[string]string
 }
 
-func NewKafkaMockCluster() (*KafkaMockCluster, error) {
-	cluster, err := kafka.NewMockCluster(brokerCount)
+func NewKafkaMockCluster(emulatorExecutablePath string) (*KafkaMockCluster, error) {
+	cmd := exec.Command("java", "-jar", emulatorExecutablePath)
```

Comment:
Please consider checking cmd.Err before going further

Reply:
We are building containers with `go1.18`. `os/exec.Cmd` doesn't have `Err` field in that version of Go: https://cs.opensource.google/go/go/+/refs/tags/go1.18.9:src/os/exec/exec.go;l=57